galsim/config/image_scattered.py

```diff
@@ -101,7 +92,8 @@ def buildImage(self, config, base, image_num, obj_num, logger):
         base['current_image'] = full_image
 
         if 'image_pos' in config and 'world_pos' in config:
-            raise AttributeError("Both image_pos and world_pos specified for Scattered image.")
+            raise galsim.GalSimConfigError(
```

Comment:
I could at least go to GalSimConfigValueError I guess.  But yes, I wanted GalSimConfigError to be a base class for all errors related to parsing a config dict.  That's a kind of user error that someone might want to guard against in their code, since it's possible that the coder doesn't have control over what config dict will be parsed.